File: scripts/plants_export.py
#!/usr/bin/env python3
import json, csv, sys
sys.path.insert(0, 'cg-field-app')
from plants import PLANTS

with open('plants_export.csv', 'w', newline='') as f:
    writer = csv.writer(f)
    writer.writerow(['Botanical','Common','Size','Target','Aggression','Type','Fertilize'])
    for p in PLANTS:
        writer.writerow([p.get('botanical',''), p.get('common',''), p.get('size',''), 
                         p.get('target',''), p.get('aggression',''), p.get('type',''), 
                         p.get('fertilize','')])

print(f'Exported {len(PLANTS)} plants to plants_export.csv')

File: scripts/plants_export.js
const fs = require('fs');
const { PLANTS } = require('./plants.js');

const header = 'Botanical,Common,Size,Target,Aggression,Type,Fertilize';
const rows = [header];

PLANTS.forEach(p => {
  const escape = v => '"' + (v || '').replace(/"/g, '""') + '"';
  rows.push([escape(p.botanical), escape(p.common), escape(p.size), escape(p.target), escape(p.aggression), escape(p.type), escape(p.fertilize)].join(','));
});

fs.writeFileSync('plants_export.csv', rows.join('\n'));
console.log('Exported', PLANTS.length, 'plants to plants_export.csv');